panedrlite/panedrlite/panedr.py

```diff
@@ -188,18 +202,23 @@ def do_eheader(self, nre_test):
         assert fr.nblock >= 0
         if ndisre != 0:
             if file_version >= 4:
-                raise ValueError("Distance restraint blocks in old style in new style file")
+                raise ValueError("Distance restraint blocks "
+                                 "in old style in new style file")
             fr.nblock += 1
-        # Frames could have nre=0, so we can not rely only on the fr.nre check
+        # Frames could have nre=0,
+        # so we can not rely only on the fr.nre check
         if (nre_test >= 0
             and ((fr.nre > 0 and fr.nre != nre_test)
                  or fr.nre < 0 or ndisre < 0 or fr.nblock < 0)):
-            bWrongPrecision = True
+            # wrong precision
             return
```

Comment:
We are raising ValueErrors for most of the other things that could go wrong, so I'm thinking if it might make sense to do the same here or at least raise a warning when the precision is wrong

Reply:
Turns out this is actually dead code.